src/commands/MurderNistery/MurderNistery.js

```diff
@@ -0,0 +1,288 @@
+const { DMChannel } = require('discord.js');
+const BaseCommand = require('../../utils/structures/BaseCommand');
+const { sleep, State } = require('./utils');
+const playerEmojis = ['😴', '😷', '🤥', '🤓', '🤔', '❌'];
+
+module.exports = class TestCommand extends BaseCommand {
+  constructor() {
+    super('nistery', 'game', []);
+  }
+
+  async run(client, message, args) {
+    if (args[0] && args[0] === "start") {
+        this.gameStart(client, message.channel);
+        return;
+    }
+
+    if (client.nistery) {
+        message.channel.send("There's already a game in progress. You're really eager to kill your friends, aren't you? 🧐");
+        return;
+    }
+
+    this.gameIntro(client, message);
+  }
+
+  async gameIntro(client, message) {
```

Comment:
Since this file is so big, maybe you could separate the different game parts in different files? I think it will become easier to read and to maintain in the future